resources/js/views/settings/Application.vue

```diff
@@ -0,0 +1,471 @@
+                    type="text"
+                    :disabled="isBusy || viewOnly"
+                    :state='fieldState("name")'
+      >
+      </b-form-input>
+
+      <template slot='invalid-feedback'>
+        <div v-html="fieldError('name')"></div>
+      </template>
+    </b-form-group>
+
+    <!--Favicon Settings-->
+    <b-form-group
+      label-for="application-favicon-input"
+      :state='(fieldState("favicon") === null && fieldState("favicon_file") === null) ? null : false'
+    >
+
+      <template v-slot:label>
+        {{ $t('settings.application.favicon.title') }}
+      </template>
+
+      <b-row class="my-3" align-v="center">
+        <b-col sm="6" lg="3" class="text-center">
+          <b-img
+            v-if="uploadFaviconFileSrc!==null || settings.favicon!==null"
+            :src="uploadFaviconFileSrc ? uploadFaviconFileSrc : settings.favicon"
+            class="my-2"
+            rounded="0"
+            alt="application-favicon-preview"
+            width="32"
+            height="32"
+            fluid
+          >
+          </b-img>
+        </b-col>
+        <b-col sm="6" lg="9">
+          <b-form-text v-if="!uploadFaviconFile">{{ $t('settings.application.favicon.urlTitle') }}</b-form-text>
+          <b-input-group>
+            <b-form-input
+              id="application-favicon-input"
+              v-if="!uploadLogoFile"
+              :placeholder="$t('settings.application.favicon.hint')"
+              v-model="settings.favicon"
+              :disabled="isBusy || viewOnly"
+              class="my-2"
+              :state='fieldState("favicon")'
+            >
+            </b-form-input>
+          </b-input-group>
+          <b-form-text v-if="!viewOnly">{{ $t('settings.application.favicon.uploadTitle') }}</b-form-text>
+          <b-input-group v-if="!viewOnly">
+            <b-form-file
+              id="application-favicon-form-file"
+              :state='fieldState("favicon_file")'
+              :disabled="isBusy"
+              :browse-text="$t('app.browse')"
+              :placeholder="$t('settings.application.favicon.selectFile')"
+              v-model="uploadFaviconFile"
+              accept="image/x-icon"
+            >
+            </b-form-file>
+            <template #append v-if="uploadFaviconFile">
+              <b-button variant="danger" @click="uploadFaviconFile = null">
+                <b-icon-x></b-icon-x>
+              </b-button>
+            </template>
+          </b-input-group>
+        </b-col>
+      </b-row>
+
+      <template slot='invalid-feedback'>
+        <div v-html="fieldError('favicon')"></div>
+        <div v-html="fieldError('favicon_file')"></div>
+      </template>
+    </b-form-group>
+
+      <!--Logo Settings-->
+      <b-form-group
+        label-for="application-logo-input"
+        :state='(fieldState("logo") === null && fieldState("logo_file") === null) ? null : false'
+      >
+
+        <template v-slot:label>
```

Comment:
Why we need the extra template? The label could be set in the form group through an attribute!